src/foam/core/AbstractPropertyInfo.java

```diff
@@ -48,12 +49,11 @@ public void toCSV(foam.lib.csv.Outputter outputter, Object value) {
 
   @Override
   public String createStatement() {
-    return getName();
+    return getName().toLowerCase();
```

Comment:
Why is this needed? Isn't Postgres case-insensitive?

Reply:
PostgreSQL converts identifiers to lowercase if they are not double-quoted. So instead of double quoting all table names & column names I figured it would be easier to just convert to lowercase instead.

Reply:
Okay, then can we not include this method in the AbtractPropertyInfo and have a custom implementation generated for each one specificiall? This will let us convert to lowercase at compile time and so we'll be faster and generate less garbage. Can be done in future PR.